src/test/java/org/folio/controller/KafkaEventListenerTest.java

```diff
@@ -93,7 +93,7 @@ private static void publishEvent(String topic, String payload) {
 
   private static void waitForOffset(String topic, String consumerGroupId, int expectedOffset) {
     Awaitility.await()
-      .atMost(30, TimeUnit.SECONDS)
+      .atMost(60, TimeUnit.SECONDS)
```

Comment:
Is this really necessary?

Reply:
Kafka event consumption test is still failing on Jenkins by timeout occasionally. Increasing this value seems to help a bit.